src/components/views/ScreeningsFilters.vue

```diff
@@ -0,0 +1,130 @@
+<script>
+import { defineComponent } from 'vue';
+import SectionTitle from '@/components/common/SectionTitle.vue';
+import SectionSubtitle from '@/components/common/SectionSubtitle.vue';
+import TheContainer from '@/components/common/TheContainer.vue';
+import AppLabel from '@/components/common/AppLabel.vue';
+import AppButton from '@/components/common/AppButton.vue';
+import AppSelect from '@/components/common/AppSelect.vue';
+import DatePicker from '@/components/features/DatePicker.vue';
+
+export default defineComponent({
+  components: { SectionTitle, SectionSubtitle, TheContainer, AppLabel, AppButton, AppSelect, DatePicker },
+  data() {
+    return {
+      isActive: false,
+      activeDay: 'Today',
+    }
+  },
+  computed: {
+    todaysDate() {
+      return new Date().toLocaleDateString('en-GB');
+    },
+    todaysDayName() {
+      return new Date().toLocaleString('en-US', {weekday: 'long'});
+    },
+    nextDayNames() {
+      const dayDigit = new Date().getDay() + 1;
+      const days = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
+      let nextDays = ['Today'];
+      for (let i=0; i<5; i++) {
+        let day = dayDigit + i;
+        console.log(day)
```

Comment:
As a good practice - try not to commit `console.log`s with your code 😁

Reply:
What a cringe... 🙄😄